rfcs/testdriver_set_time_zone.md

```diff
@@ -0,0 +1,21 @@
+# RFC 73 : Add testdriver.js support for the `timeZone` WebDriver command.
+
+## Summary
+Add testdriver.js support for the `timeZone` WebDriver extension, which will
+allow setting the current time zone in the browser. This extension is specced
+as part of the [HTML spec](https://github.com/whatwg/html/pull/3047).
+
+## Details
+Changing the time zone is not available via normal web APIs. To test the effect
+of changing time zone on the `ontimezonechange` event there needs to be a way
```

Comment:
Terminology nit:

```suggestion
of changing time zone on the `timezonechange` event there needs to be a way
```

`timezonechange` is the event. `ontimezonechange` is the corresponding event handler. https://annevankesteren.nl/2011/08/on-events